swift-swiftLint.md

```diff
@@ -0,0 +1,134 @@
+# Introduction
+
+[SwiftLint](https://github.com/realm/SwiftLint/blob/master/README.md) is a tool by Realm to enforce Swift style and conventions. SwiftLint hooks into Apple’s own SourceKit framework to parse your Swift code, which means it supports the full range of Swift syntax and remains up to date as Swift continues to evolve. It is based on guidelines from the Swift style guide.
+<br /><br />
+# Prerequisite
+
+* [Swift 5.6](https://www.swift.org/blog/swift-5.6-released/)
+  <br /><br />An existing application or new application developed using Swift.
+
+
+* [XCode 13.3](https://developer.apple.com/documentation/Xcode-Release-Notes/xcode-13_3-release-notes)
+* [CocoaPod](https://cocoapods.org)
+
+# Installation
+
+### Using [Homebrew](http://brew.sh/):
+
+```
+brew install swiftlint
+```
+
+### Using [CocoaPods](https://cocoapods.org):
```

Comment:
We should standadize to use 1 thing. Either `homebrew` or `cocapods`.

If they are like `npm` and `yarn`. We decided to go with just `npm`, then you need to pick the good and reliable one and only use that.

Reply:
- Removed Carthage option.
- Recommended CocoaPods